### PR TITLE
docs(): documented fix for can't find .env

### PR DESCRIPTION
**Pull request by god-of-js**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Documentation

## Context

- [ ] Closes first installation issue - "ERROR: Couldn't find env file: /Users/admin/Desktop/codes/randoms/test/nft-gallery/.env" (No issue was created..... Sorry....);
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?
No, just a quick documentation addition as it didn't have an issue.

- [ ] My DOT address: 12AVSmE1QguooF6HCMzJxTiYcWaaFb6Vgm7YixmMRyE8S9pV [Payout](https://canary.kodadot.xyz/dot/transfer/?target=12AVSmE1QguooF6HCMzJxTiYcWaaFb6Vgm7YixmMRyE8S9pV)

### <samp>🤖 Generated by Copilot at f53f55a</samp>

Added a troubleshooting tip for docker build errors in `content/blog/first-time.md`. The tip helps users fix a possible issue with the `.env` file.

### <samp>🤖 Generated by Copilot at f53f55a</samp>

> _Troubleshoot `.env`_
> _Docker build may fail in spring_
> _Fix with simple step_

### CODE REVIEW

File: content/blog/first-time.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ docker-compose up --build
 docker ps
 ```
 
+- Getting couldn't find .env file error?
+
+In some cases, you may get an error similar to `ERROR: Couldn't find env file: /Users/admin/Desktop/codes/randoms/test/nft-gallery/.env`; This is because docker may be looking for a `.env` file when there is none. To fix this issue, create an empty `.env` file then build the docker image again.
```
Comment (roiLeo) [inline content/blog/first-time.md:59]: I would change/hide this full path.

**Review by roiLeo — COMMENTED** [review]
We could add an `.env.example` with the available env variables

**Comment by god-of-js** [comment]
> We could add an `.env.example` with the available env variables

Can you send me the `.env` variables?

File: content/blog/first-time.md (hunk 1 above)
Comment (god-of-js) [reply content/blog/first-time.md]: Done

**Comment by roiLeo** [comment]
> Can you send me the `.env` variables?

Search for `process.env.` string key

**Comment by god-of-js** [comment]
> @god-of-js follow PR template please, otherwise I will close this PR
> 
> https://github.com/kodadot/nft-gallery/blob/main/.github/PULL_REQUEST_TEMPLATE/QUICK_PULL_REQUEST_TEMPLATE.md

Done.

**Comment by god-of-js** [comment]
> Search for `process.env.` string key

Done that, I can see a couple of places it's been used, but couldn't find where the variables are declared.

Is there a .env for the dev environment?

**Comment by roiLeo** [comment]
> Done that, I can see a couple of places it's been used, but couldn't find where the variables are declared.

just add empty string

> Is there a .env for the dev environment?

Yes and one for production too

**Comment by yangwao** [comment]
@god-of-js feel free drop your payout address, legit bug report

**Comment by god-of-js** [comment]
> @god-of-js feel free drop your payout address, legit bug report

Aye Aye captain

**Comment by god-of-js** [comment]
> > Done that, I can see a couple of places it's been used, but couldn't find where the variables are declared.
> 
> just add empty string
> 
> > Is there a .env for the dev environment?
> 
> Yes and one for production too

Okay, done

**Comment by yangwao** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 5.03 USD/DOT ~ 1.988 $DOT
🧗 [12AVSmE1QguooF6HCMzJxTiYcWaaFb6Vgm7YixmMRyE8S9pV](https://polkadot.subscan.io/account/12AVSmE1QguooF6HCMzJxTiYcWaaFb6Vgm7YixmMRyE8S9pV)
🔗 [0xf25921d974ee8a0ebc074c349bcb2381a92601e3c1b215ddb4007271649cfdc9](https://polkadot.subscan.io/extrinsic/0xf25921d974ee8a0ebc074c349bcb2381a92601e3c1b215ddb4007271649cfdc9)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**